combat-trainer.lic

```diff
@@ -1764,7 +1764,7 @@ class AttackProcess
     @fatigue_regen_action = settings.fatigue_regen_action
     echo("  @fatigue_regen_action: #{@fatigue_regen_action}") if $debug_mode_ct
 
-    @is_empath = DRStats.empath? && !game_state.construct_mode?
```

Comment:
Did you test this? The `construct_mode?` predicate is defined in GameState - I'm confused by this change